scripts/vsc-venv-wrapper/vsc-venv.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+
+VENV_NAME="venv-${VSC_OS_LOCAL}-${VSC_ARCH_LOCAL}"
+VENV_LOCATION=$(realpath -m "venvs/$VENV_NAME") # full path of venv
+
+usage() {
+  echo "Usage: $0 {create <optional: modules>|install <requirements>|activate}"
+  echo ""
+  echo "Commands:"
+  echo "  create [modules_script]  Create a virtual environment at $VENV_LOCATION."
+  echo "                           Optionally, load modules from the specified script before creating the environment."
+  echo "  install <requirements>   Install packages from the specified requirements file."
+  echo "                           The virtual environment must be activated before running this command."
+  echo "  activate                 Activate the virtual environment. You need to source this command."
+  echo ""
+  echo "Example Usage:"
+  echo "  $ $0 create my_modules.sh"
+  echo "  $ source $0 activate"
+  echo "  $ $0 install requirements.txt"
+  echo "  $ python my_script.py"
+  echo "  $ deactivate"
```

Comment:
for consistency, i would make this `$0 deactivate`. and you never know you want to gather some more metadata on deactivation, eg the loaded modules on deactivation.